server/main.lua

```diff
@@ -44,9 +44,20 @@ end)
 lib.addCommand('startshow', {
     help = locale('start_show_desc'),
     params = {
-        {name = "Location", help = locale('start_show_param_hint'), type = 'string'}
+        { name = "Location", help = locale('start_show_param_hint'), type = 'string' }
     },
     restricted = 'group.admin'
 }, function(source, args, raw)
-    TriggerClientEvent('qbx_fireworks:client:startShow', source, args.Location)
+    local show = args.Location
+    if not show then return end
+    if not config.shows[show] then return end
+    for i = 1, #config.shows[show].fireworks do -- Sequences
+        local firework = config.shows[show].fireworks[i]
+        local fireworkAsset = firework.asset
+        local fireworkCoords = firework.coords
+        local fireworkHeight = firework.height
+        local fireworkWait = firework.wait
```

Comment:
Not sure there is any readability gain by storing these variables

Reply:
removed variables